scripts/distribution/builder.Dockerfile

```diff
@@ -77,8 +77,8 @@ ENV DISTRIBUTION_CLIENT=true
 ENV ENABLE_TERM_HANDLER=true
 
 RUN apt-get update && apt-get install -y unbound curl netbase ca-certificates supervisor nginx libnuma1 libtinfo6 libpq-dev liblmdb-dev jq
-RUN curl -L https://getenvoy.io/cli | bash -s -- -b /usr/local/bin
-RUN getenvoy fetch standard:1.17.0
+RUN curl -L https://getenvoy.io/install.sh | bash -s -- -b /usr/local/bin
+RUN getenvoy use 1.18.3
```

Comment:
I don't think the versions should just be changed without explicitly planning to do so, and planning to test the resulting version bump.

Reply:
The new release of `getenvoy` doesn't work with older versions of `envoy`. If the new version works (which it seems to) then I think it would be better just to use it. Otherwise, we need a better way of getting a fixed version of envoy.

Reply:
Yes, as Thomas says it's unfortunate. 

We could perhaps use a docker image e.g., `envoyproxy/envoy:v1.17.0 ` if we really want to stay on the same version. But that would also be quite a change to the distribution itself.

Reply:
We can't use that directly because we make one image with all necessities in it. Changing this setup would require substantial changes in how this image is used.

Additionally, copying selected binaries from one docker image to another, with unclear and undocumented dependencies is a horrible, hacky, and fragile practice we should not do.

Reply:
Yep, that is what I meant with the "But that would also be quite a change to the distribution itself." comment. :-) 

And yes I agree, that we should find a proper solution to this.

Reply:
So I suppose the least shitty solution is to update the version now. I'll create an issue to fix this as well.

Could you please test this locally to make sure the entire node dashboard works as it should? Then we can merge.

Reply:
I did that already.

Reply:
> Additionally, copying selected binaries from one docker image to another, with unclear and undocumented dependencies is a horrible, hacky, and fragile practice we should not do.

I disagree with this. It's a bit hacky, but this might be the least fragile way of doing it (that doesn't involve building it from source) since it is very unlikely that a particular docker image will change or disappear.

Reply:
It's not just hacky, it only works "by accident" while the shared library dependencies in those images are close enough.

If some program is built in an image with libc version Y and then copied somewhere where libc version X < Y is installed it likely won't work. 

Unless these are explicitly pinned and care is taken to either build fully static binaries (that means no libc dependencies) or use the same base image then it will break.